cpp/src/arrow/python/datetime.h

```diff
@@ -157,6 +157,22 @@ inline int64_t PyDelta_to_ns(PyDateTime_Delta* pytimedelta) {
   return PyDelta_to_us(pytimedelta) * 1000;
 }
 
+ARROW_PYTHON_EXPORT
```

Comment:
Do these need to be exported?

Reply:
Well, the other utility functions are exported as well.